docs/applications/container-repositories/02-github.md

```diff
@@ -0,0 +1,137 @@
+
+After you publish your application, you'll need to create a package. You can do this by dragging your application out from the bundle bar in the Massdriver canvas. Fill in the values of your application and click **Deploy**.
+
+Copy and save the package slug by hovering over the name of your application on the canvas. This will be used in the next step.
+
+## Set secrets and vars
+
+Once you've published your application, you'll need to set the following secrets and vars in your GitHub repository:
+
+| Name | Description | Type | Notes |
+| --- | --- | --- | --- |
+| `MASSDRIVER_ORG_ID` | Your Massdriver organization ID | secret | Your Organization ID can be found by hovering over your org logo in the sidebar |
+| `MASSDRIVER_API_KEY` | Your Massdriver API key | secret | [Service Accounts](/platform/service-accounts) |
+| `AUTHENTICATION_ARTIFACT_ID` | The ID of the authentication artifact in Massdriver | secret | Select your cloud credential from the Artifacts page in Massdriver, then click **Copy Artifact ID** |
+| `NAMESPACE` | The namespace of your build | variable | If it does not exist paired with the `IMAGE_NAME`, then it will be created for you |
+| `IMAGE_NAME` | The image name of your build | variable | If it does not exist paired with the `NAMESPACE`, then it will be created for you |
+| `REGION` | The region where your cloud container repository is located | variable | Must be a valid cloud region. For example: `eastus` for Azure, or `us-west-1` for AWS. |
+
+## Workflow file
+
+:::note
+
+This GitHub Action will create a new cloud container repository with the following name format: `massdriver<region>`. The region will be the same as the region you set in the `REGION` variable. If the registry already exists, the action will use it.
+
+:::
+
+To set up the GitHub Action, create a new file named `deploy.yaml` in the `.github/workflows` directory of your GitHub repository. You can use this workflow below:
+
+```yaml title=".github/workflows/deploy.yaml"
+name: Deploy to Massdriver
+on:
+  push:
+    branches: [main]
+
+jobs:
+  push_and_deploy:
+    runs-on: ubuntu-latest
+    env:
+      MASSDRIVER_ORG_ID: ${{ secrets.MASSDRIVER_ORG_ID }}
+      MASSDRIVER_API_KEY: ${{ secrets.MASSDRIVER_API_KEY }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Massdriver CLI
+        uses: massdriver-cloud/actions@v3
+      - name: Push Image
+        uses: massdriver-cloud/actions/image_push@v3
+        with:
+          namespace: ${{ vars.NAMESPACE }}
+          image-name: ${{ vars.IMAGE_NAME }}
+          artifact: ${{ secrets.AUTHENTICATION_ARTIFACT_ID }}
+          region: ${{ vars.REGION }}
+          image-tag: ${{ github.sha }}
+          build-context: ./
+      - name: Publish Bundle 
+        uses: massdriver-cloud/actions/bundle_publish@v3
+        with:
+          build-directory: ./
```

Comment:
Can be omitted, is the default.

Reply:
I think a build directory of `./` is probably very rare. Most people probably store the configs in a `massdriver` or `mass` directory or something. We should probably note that here instead of `./`, like `<path to massdriver configuration directory>`.